hadoop-hdds/interface-server/src/main/proto/ScmServerDatanodeHeartbeatProtocol.proto

```diff
@@ -303,6 +315,8 @@ message SCMCommandProto {
   optional ReplicateContainerCommandProto replicateContainerCommandProto = 6;
   optional CreatePipelineCommandProto createPipelineCommandProto = 7;
   optional ClosePipelineCommandProto closePipelineCommandProto = 8;
+  optional FinalizeNewLayoutVersionCommandProto
+  finalizeNewLayoutVersionCommandProto = 9;
```

Comment:
Why not use a single line: `optional FinalizeNewLayoutVersionCommandProto=9`?

Reply:
> 80 columns

Reply:
Ah I see. Didn't know Ozone also enforce that in proto files (very common that message name is very long).